Assets/MRTK/Examples/Demos/HandTracking/Scripts/DemoRuntimeMenuBuilder.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.UI;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Examples.Demos
+{
+    /// <summary>
+    /// Demo script that dynamically adds and removes buttons from a GridObjectCollection.
+    /// </summary>
+    [RequireComponent(typeof(GridObjectCollection))]
+    public class DemoRuntimeMenuBuilder : MonoBehaviour
+    {
```

Comment:
Can you move this script to MRTK/Examples/Experimental/GridObjectCollectionFitter/Scripts